### ISSUE TITLE
Update link styling - underlined and blue for perceivability and recognition

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/resources-ressources#83]
Consider whether the same styling should be applied to breadcrumbs, language toggle, footer, etc. as well.

**Comment by anikbrazeau** [comment]
--> Standardize header and footer links
--> Document decisions about buttons and links styling

**Comment by anikbrazeau** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [High-priority GitHub issues before launch](https://trello.com/c/rVY120dI/325-high-priority-github-issues-before-launch)
